backend/src/api/fishtanks/fishtankInteractions/index.js

```diff
@@ -0,0 +1,20 @@
+const { Router } = require('express');
+
+const router = new Router({ mergeParams: true });
+
+/**
+ * @api {get} /fishtanks/:id/interactions Get fishtank interactions
+ * @apiName GetFishtankInteractions
+ * @apiGroup Static Pages
+ * @apiDescription This URL displays data about a given fishtank interactions
+ *
+ * @apiParam  {Integer} id Number identifying the fishtank
+ * @apiParamExample  {String} Request-Example:
+ id: 1
+ *
+ * @apiSuccessExample {html} Success-Response:
```

Comment:
Probably return `{json}` instead of `{html}`, but since the example response doesn't look like the correct one it's probably not worth fixing now

Reply:
Yeah definitely already fixed some of them in the develop branch, just doesn't feel right to fix the documentation for placeholder routes :grin: